corehq/apps/export/forms.py

```diff
@@ -1,139 +1,336 @@
-            for app in apps if hasattr(app, 'modules')
-            for module in app.modules
-        ]
-        self.fields['form'].choices = [
-            (form.get_unique_id(), form.name)
-            for app in apps
-            for form in app.get_forms()
-        ]
+    def __init__(self, *args, **kwargs):
+        super(CreateFormExportTagForm, self).__init__(*args, **kwargs)
 
         self.helper = FormHelper()
-        self.helper.form_id = "create-export-form"
-        self.helper.form_class = "form-horizontal"
+        self.helper.form_tag = False
+        self.helper.label_class = 'col-sm-2'
+        self.helper.field_class = 'col-sm-10'
 
         self.helper.layout = crispy.Layout(
-            crispy.Fieldset(
-                _('Select Form'),
-                crispy.Field(
-                    'application',
-                    data_bind='value: appId',
-                ),
-                crispy.Div(
-                    crispy.Field(
-                        'module',
-                        data_bind=(
-                            "options: moduleOptions, "
-                            "optionsText: 'text', "
-                            "optionsValue: 'value', "
-                            "value: moduleId"
-                        ),
-                    ),
-                    data_bind="visible: appId",
-                ),
-                crispy.Div(
-                    crispy.Field(
-                        'form',
-                        data_bind=(
-                            "options: formOptions, "
-                            "optionsText: 'text', "
-                            "optionsValue: 'value', "
-                            "value: formId"
-                        ),
-                    ),
-                    data_bind="visible: moduleId",
-                ),
+            crispy.Field(
+                'application',
+                placeholder=_("Select Application"),
+                ng_model="createForm.application",
+                ng_change="updateModules()",
+                ng_required="true",
             ),
-            crispy.Div(
-                FormActions(
-                    crispy.ButtonHolder(
-                        crispy.Submit(
-                            'create_export',
-                            _('Next'),
-                        ),
-                    ),
-                ),
-                data_bind="visible: formId",
+            crispy.Field(
+                'module',
+                placeholder=_("Select Module"),
+                ng_model="createForm.module",
+                ng_disabled="!createForm.application",
+                ng_change="updateForms()",
+                ng_required="true",
+            ),
+            crispy.Field(
+                'form',
+                placeholder=_("Select Form"),
+                ng_model="createForm.form",
+                ng_disabled="!createForm.module",
+                ng_required="true",
```

Comment:
I don't think this should block this PR. Also no idea that that existed. Note that that will have to be rewritten in angular.